ZAPDTR/ZAPD/ZDisplayList.cpp

```diff
@@ -15,6 +15,124 @@
 #include "WarningHandler.h"
 #include "gfxd.h"
 
+
```

Comment:
Is moving this from a .cpp to a .h the correct thing to to here?

Previously these would not be defined in anything that included ZDisplayList.h, but now they will be. Is that intended? 

(Note, I'm not a C++ expert, so some of this review might be ignorant questions)

Reply:
It's the contrary, I'm moving these macros from a .h to a .cpp to avoid conflicts in any translation unit that includes both ZDisplayList.h and gbi.h. (Blame MSVC's stupid preporocessor for not giving an error when this occurs.)